lib/luaxdp.c

```diff
@@ -106,9 +106,15 @@ __bpf_kfunc int bpf_luaxdp_run(char *key, size_t key__sz, struct xdp_md *xdp_ctx
 
 __diag_pop();
```

Comment:
I think we should also add this..
```suggestion
#if (LINUX_VERSION_CODE >= KERNEL_VERSION(6, 9, 0))
__bpf_kfunc_end_defs();
#else
__diag_pop();
#endif
```

Reply:
This should be for >= 6.4.0, right?

Reply:
Yup, you're right!